pages/room/index.js

```diff
@@ -51,23 +62,22 @@ const CreateRoom = ({ spotifyAuthURL, spotifyCode, newSession }) => {
 
   const createNewSpace = async () => {
     setRoomIsLoading(true);
-    const id = uuid();
-
+    const spaceId = uuid();
```

Comment:
the import of uuid() is from v1. v1 isn't great since it's based on time. NodeJS is single threaded so I'm not sure if this will  actually cause errors where we might get 2 of the same ids. Either way, just to be safe, can you change uuid import from v1 to v4?